src/CodeGen_OpenGL_Dev.cpp

```diff
@@ -250,6 +250,14 @@ void CodeGen_GLSL::visit(const FloatImm *op) {
     id = oss.str();
 }
 
+void CodeGen_GLSL::visit(const IntImm *op) {
+    print_assignment(op->type, print_type(op->type) + "(" + std::to_string(op->value) + ")");
```

Comment:
Should we just change the base CodeGen_C to do it this way? Seems like it shouldn't hurt and it would be simpler overall.